scipy/optimize/tests/test__sda.py

```diff
@@ -0,0 +1,120 @@
+# Simulated dual annealing unit tests implementation.
+# Copyright (c) 2017 Sylvain Gubian <sylvain.gubian@pmi.com>,
+# Yang Xiang <yang.xiang@pmi.com>
+# Author: Sylvain Gubian, PMP S.A.
+##############################################################################
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
```

Comment:
This license comment is wrong, should be licensed under BSD. However best to remove all license details in individual files, except possibly (if you prefer) a simple `License: BSD`. We prefer to cover all files by the main SciPy LICENSE file.

Reply:
Removed. Thanks, I forgot to remove from the unit tests file.